<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gp_novavenda</class>
 <widget class="QDialog" name="gp_novavenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txt_qtde">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_codproduto">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>30</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listaprodutos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>691</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantidade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>34</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CÃ³digo do Produto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor Total:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_totalvenda">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>R$</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_excluirprod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Excluir </string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_editarprod">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar </string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_finalizarvenda">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>324</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
background-color: rgb(66, 133, 244);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Finalizar Venda</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pesquisarprod">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
